File: find_sections.py
import re

path = r"C:\Users\rdpadmin\.gemini\antigravity\scratch\orcasagent-site\index.html"
with open(path, "r", encoding="utf-8") as f:
    lines = f.readlines()

print("=== Sections & IDs ===")
for i, line in enumerate(lines, 1):
    # look for section tags, id attributes, or main containers
    if "<section" in line or "id=" in line or "class=\"chamber\"" in line or "class=\"card\"" in line:
        # extract id, class, or section tag if present
        cleaned = line.strip()
        if len(cleaned) > 120:
            cleaned = cleaned[:117] + "..."
        print(f"Line {i:5d}: {cleaned}")

File: find_sections.js
const fs = require('fs');
const path = 'C:\\Users\\rdpadmin\\.gemini\\antigravity\\scratch\\orcasagent-site\\index.html';

const content = fs.readFileSync(path, 'utf8');
const lines = content.split('\n');

console.log('=== Sections & IDs in index.html ===');
lines.forEach((line, i) => {
    const lineNum = i + 1;
    if (line.includes('<section') || line.includes('id=') || line.includes('class="chamber"') || line.includes('class="card"')) {
        let cleaned = line.trim();
        if (cleaned.length > 120) {
            cleaned = cleaned.substring(0, 117) + '...';
        }
        console.log(`Line ${lineNum.toString().padStart(5, ' ')}: ${cleaned}`);
    }
});
